modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,124 @@
+----
+. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container: 
++
+[source,terminal]
+----
+$ end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
+$ sas=`az storage container generate-sas -n ${CONTAINER_NAME} --account-name ${STORAGE_ACCOUNT_NAME} --https-only --permissions dlrw --expiry $end -o tsv`
+----
+. Copy the {op-system} VHD into the storage container:
++
+[source, terminal]
+----
+$ az storage blob copy start --account-name ${STORAGE_ACCOUNT_NAME} --sas-token "$sas" \
+ --source-uri "${RHCOS_VHD_ORIGIN_URL}" \
+ --destination-blob "${BLOB_NAME}" --destination-container ${CONTAINER_NAME}
+----
++
+You can check the status of the copying process with the following command:
++
+[source,terminal]
+----
+$ az storage blob show -c ${CONTAINER_NAME} -n ${BLOB_NAME} --account-name ${STORAGE_ACCOUNT_NAME} | jq .properties.copy
+----
++
+.Example output
+[source,terminal]
+----
+{
+ "completionTime": null,
+ "destinationSnapshot": null,
+ "id": "1fd97630-03ca-489a-8c4e-cfe839c9627d",
+ "incrementalCopy": null,
+ "progress": "17179869696/17179869696",
+ "source": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-411.86.202207130959-0-azure.aarch64.vhd",
+ "status": "success", <1>
+ "statusDescription": null
+}
+----
++
+<1> If the status parameter displays the `success` object, the copying process is complete. 
+ 
+. Create an image gallery using the following command:
++
+[source,terminal]
+----
+$ az sig create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME}
+----
+Use the image gallery to create an image definition. In the following example command, `rhcos-arm64` is the name of the image definition. 
++
+[source,terminal]
+----
+$ az sig image-definition create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME}--gallery-image-definition rhcos-arm64 --publisher RedHat --offer arm --sku arm64 --os-type linux --architecture Arm64 --hyper-v-generation V2
+----
+. To get the URL of the VHD and set it to `RHCOS_VHD_URL` as the file name, run the following command: 
++
+[source,terminal]
+----
+$ RHCOS_VHD_URL=$(az storage blob url --account-name ${STORAGE_ACCOUNT_NAME} -c ${CONTAINER_NAME} -n "${BLOB_NAME}" -o tsv)
+----
+. Use the `RHCOS_VHD_URL` file, your storage account, resource group, and image gallery   to create an image version:
++
+[source,terminal]
+----
+$ az sig image-version create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME}--gallery-image-definition rhcos-arm64 --gallery-image-version 1.0.0 --os-vhd-storage-account ${STORAGE_ACCOUNT_NAME} --os-vhd-uri ${RHCOS_VHD_URL}
+----
+. Your `arm64` imageID for the `resourceID` parameter is now generated: 
++
+.Example `resourceID` 
++
+[source,terminal]
+----
+/resourceGroups/${RESOURCE_GROUP}/providers/Microsoft.Compute/galleries/${GALLERY_NAME}/images/rhcos-arm64/versions/1.0.0
```

Comment:
I changed this to use `rhcos-arm` for `${IMAGE_NAME}` and `1.0.0` for `${IMAGE_VERSION}` since those are the example names in lines 103 and 115.
Made it the same for the example yaml in next procedure.

Reply:
honestly that can be templatized as well..but that's up to you.